src/helm/proxy/models.py

```diff
@@ -476,8 +476,8 @@ def engine(self) -> str:
         group="codex",
         creator_organization="OpenAI",
         name="openai/code-cushman-001",
-        display_name="Codex code-cushman-001 (12B)",
-        description="Codex model that is a stronger, multilingual version of the Codex (12B) model in the paper.",
+        display_name="code-cushman-001 (12B)",
+        description="Code model that is a stronger, multilingual version of the Codex (12B) model in the paper.",
```

Comment:
So this is not the Codex from the paper?

Reply:
The exact quote from OpenAI is
"code-cushman-001 is a stronger, multilingual version of the Codex 12B model in [Evaluating Large Language Models Trained on Code](https://arxiv.org/abs/2107.03374)."

Source: https://beta.openai.com/docs/model-index-for-researchers